.gitmodules

```diff
@@ -1,5 +1,5 @@
-[submodule "wheatley-private"]
-	path = src/wheatley-private
+[submodule "src/components/tccpp/private"]
+	path = src/components/tccpp/private
```

Comment:
I had forgotten that wheatley-private isn't just components, it's
```
algorithm/
components/
services/
test/
```

Thoughts on keeping this top-level? Maybe moving `src/components/tccpp` to `src/tccpp/components`? Not sure what's best.

Reply:
We could have bot modules, libraries of related components and their infrastructure, that can be installed in `src/modules`. And then we just load components from `src/**/components/*.ts`.